packages/noports_core/pubspec.yaml

```diff
@@ -11,22 +11,19 @@ environment:
   sdk: ">=3.0.0 <4.0.0"
 
 dependencies:
-  archive: ^3.3.8
   args: ^2.4.2
   at_chops: ^1.0.4
   at_client: ^3.0.65
-  at_commons: ^3.0.55
-  at_lookup: ^3.0.40
   at_utils: ^3.0.15
-  crypton: ^2.1.0
+  cryptography: ^2.7.0
   dartssh2: ^2.8.2
-  logging: ^1.2.0
   meta: ^1.9.1
+  openssh_ed25519: ^1.1.0
```

Comment:
openssh_ed25519 is used for converting the generated ed25519 keys to openssh format